tests/scripts/beneficiary/delete_or_suspend_account_from_csv_test.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+from pcapi.core.users.factories import FavoriteFactory
+from pcapi.core.users.factories import UserFactory
+from pcapi.models import Deposit
+from pcapi.models import User
+from pcapi.repository import repository
+from pcapi.scripts.beneficiary.delete_or_suspend_account_from_csv import run
+
+
+@pytest.mark.usefixtures("db_session")
+def test_delete_user():
+    UserFactory(email="romain.chaffal@beta.gouv.fr", isAdmin=True)
+    user_without_deposit = UserFactory(email="user_to_delete@example.com")
+    FavoriteFactory(user=user_without_deposit)
+    deposit = Deposit.query.all()
+    repository.delete(*deposit)
+    run(
+        "tests/scripts/beneficiary/users_to_delete_fixture.csv",
+        admin_email_used="romain.chaffal@beta.gouv.fr",
+    )
+    User.query.one()
```

Comment:
Le test serait sûrement plus clair avec quelque chose comme ça :

```suggestion
    assert User.query.all() == [admin]
```